onnx_tf/third_party/get_info.py

```diff
@@ -23,45 +23,45 @@
 
 
 def trim(docstring):
```

Comment:
Please exclude this file from linting. As the name of the folder suggests, modifying this file carries complicated legal implications I do not want to get involved with.